opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -139,6 +139,109 @@ class SpanKind(enum.Enum):
     CONSUMER = 4
 
 
+class StatusCanonicalCode(enum.Enum):
+    """Represents the canonical set of status codes of a finished Span, following the Standard GRPC codes. See
+    https://github.com/open-telemetry/opentelemetry-specification/blob/2dbae9a491224f1fddfa2bb05c2a1a444c623077/specification/api-tracing.md#statuscanonicalcode.
+    """
+
+    #: Not an error; returned on success.
+    OK = 0
+
+    #: The operation was cancelled, typically by the caller.
+    CANCELLED = 1
+
+    #: Unknown error. For example, this error may be returned when a Status value received from another address space belongs to an error space that is not known in this address space.
```

Comment:
These would be easier to read if you re-wrapped these lines instead of keeping the original formatting.